File: PRAKTIKUM PEMROGRAMAN I P1/MODUL 6/SOAL 3/PRAK603-2410817210002-Sebastian_Abe_Santoso.c
#include <stdio.h>
int main(){
    int baris, baris2, a, b, c, d, total;
    scanf("%d %d", &baris, &baris2);
    if (baris != baris2){
        printf("Jumlah tidak sama");
    }   else {
    int matriks[baris][baris2];
    for (a = 0; a < 2; a++){
        for (b = 0; b < baris2; b++){
            scanf("%d", &matriks[a][b]);
        }
    }   for (c = 0; c < baris; c++){
        int total = matriks[0][c] * matriks[1][c];
        printf("%d ", total);       
        }
    } return 0;
}

File: MODUL 6/SOAL 3/PRAK603-2410817210002-Sebastian_Abe_Santoso.py
baris = list(map(int, input().split()))
baris1, baris2 = baris[:2]
if baris1 != baris2:
    print("Jumlah baris harus sama")
else:
    matriks = []
    for a in range(2):  
        row = list(map(int, input().split()))
        matriks.append(row)
    for c in range(baris2):
        total = matriks[0][c] * matriks[1][c]  
        print(total, end=' ')
    print()